src/ape_accounts/_cli.py

```diff
@@ -86,17 +87,36 @@ def generate(cli_ctx, alias):
 
 
 # Different name because `import` is a keyword
-@cli.command(name="import", short_help="Add a new keyfile account by entering a private key")
+@cli.command(
+    name="import", short_help="Add a new keyfile account by entering a private key or seed phrase"
+)
+@click.option(
+    "--mnemonic", "import_from_mnemonic", help="Import a key from a mnemonic", is_flag=True
+)
+@click.option("--hdpath", "custom_hd_path", help="Input HD Path to use", is_flag=True)
```

Comment:
To get this work with values, we would need to remove `is_flag` here.
Consider if it worth also putting `default=DEFAULT_ETHEREUM_HD_PATH` here.